web_sdk/libraries.dart

```diff
@@ -0,0 +1,310 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
This file has suspicious code style and copyright year. Has it been branched from somewhere?